### PR TITLE
Highlights List + CTA full width for Mags teal

### PR DESCRIPTION
**Pull request by wilson1000-MoJ**
Toni and I had a conversation about the width of the teal components - these are meant to be full width.

This update has been tested with Chrome and IE11

### CODE REVIEW

File: src/01-highlights-list/editor.scss [hunk 1]
```diff
@@ -1,10 +1,9 @@
 @charset "UTF-8";
 /*********************************************************************************************
 
-Plugin Name: MOJ Blocks
-Block: Accordion (Gutenberg editor styles)
-Text Domain: mojblocks
+Name: Highlights List
+Author:  Ministry of Justice - Damien Wilson
```
Comment (brown-a2) [inline src/01-highlights-list/editor.scss:6]: Yes, that accordion had to go. Wonder if we still should keep reference that it is a Gutenberg editor stylesheet? And btw, I don't mind if you want to put your name on individual blocks but I'm probably not going to be updating every block I work on with my name.

File: src/02-cta/index.js [hunk 2]
```diff
@@ -76,8 +76,9 @@ registerBlockType('mojblocks/cta', {
 
         return (
             <div className={`${className}`}>
-                <div className="mojblocks-cta__heading-container">
-                    <h3 className="mojblocks-cta__heading">
+                <div className={'nhsuk-width-container'}>
```
Comment (brown-a2) [inline src/02-cta/index.js:79]: are we not removing all nhsuk class styles in our blocks?

File: src/frontend.js [hunk 3]
```diff
@@ -1,12 +1,12 @@
 /**
  * Gov UK Frontend JS
- * Queues up the Gov UK JS modules requried for our blocks
+ * Queues up the Gov UK JS modules required for our blocks
  */
```
Comment (brown-a2) [inline src/frontend.js:4]: good spot!

File: style-gutenburg.scss [hunk 4]
```diff
@@ -18,4 +18,9 @@ Text Domain: mojblocks
...
     @import 'node_modules/govuk-frontend/govuk/helpers/media-queries';
     @import 'src/**/editor.scss';
-}
\ No newline at end of file
+}
+
+/// adjust width of editor to match NHS theme
+body.block-editor-page.kt-editor-width-default div.wp-block {
+    max-width: 960px;
+}
```
Comment (brown-a2) [inline style-gutenburg.scss:26]: Not sure the plugin should be influencing the theme editor (even if it is in the backend)?

File: style.scss [hunk 5]
```diff
@@ -15,4 +15,9 @@ Text Domain: mojblocks
...
 @import 'node_modules/govuk-frontend/govuk/settings/media-queries';
 @import 'node_modules/govuk-frontend/govuk/helpers/media-queries';
-@import 'src/**/style.scss';
\ No newline at end of file
+@import 'src/**/style.scss';
+
+/// hack: prevent a horizontal scrollbar in IE11
+main {
+    overflow-x: hidden;
+}
```
Comment (brown-a2) [inline style.scss:23]: I wonder if we should have a dedicated IE11 sass file or we just go with adding IE hacks in here...

File: style-gutenburg.scss (hunk 4 above)
Comment (wilson1000-MoJ) [reply style-gutenburg.scss]: Thanks mate. Removed for now without an additional solution. The editor styling is now confusing but we can change that.

File: src/02-cta/index.js (hunk 2 above)
Comment (wilson1000-MoJ) [reply src/02-cta/index.js:79]: All done now :)

File: style.scss (hunk 5 above)
Comment (wilson1000-MoJ) [reply style.scss:29]: We should yes. I think we should revisit this conversation with the accessibility convo we need to have. What do you think?

**Comment by RobLoweMOJ** [comment]
@brown-a2 @wilson1000-MoJ There is a conflict with this PR and the accordion block i believe the new Gov UK styles are overriding the style sin the block. We may need to be more specific on what govuk classes we import.

**Comment by brown-a2** [comment]
I think it has to do with the introduction of the "govuk.scss" file structure?

**Pull request merged**